### DISCUSSION TITLE
ipv6 overrighting Ip4

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2336] [Help with using Postal]
i am encountering an issue here please, i noticed that my ip6 is the one used in sending my emails even though they are not configured on postal. i configured only ip4 and still, it is ip6 that postal uses to send the emails which end up being rejected. HOW CAN I FIX THIS PLEASE??

**Comment by willpower232** [comment]
Postal binds itself to the local IPv6 address by default giving it access to both

https://github.com/postalserver/postal/blob/9f4ef8f57a839c5529b4f00a36b832740386b4ed/config/postal.defaults.yml#L67

If you change that to a specific IP address (or 0.0.0.0), it may resolve your issue. Otherwise you may have to disable IPv6 on your server which is definitely not a recommended long term solution but others here have done it.

**Reply by Itseric100** [reply]
Uhm I am confused right now, I installed the latest version of postal and it has only postal.yml and not postal.defaults.yml. Do I need to create this file separately or how do I go about it pelase. Thanks

**Reply by willpower232** [reply]
That is the file of default values which you override with the postal.yml file you do see.

You should be able to locate the `smtp_server` heading in your file so can add `bind_address` after it referring to your IPv4 address.
